backend/src/requirements.txt

```diff
@@ -2,4 +2,7 @@ ariadne==0.22
 Flask==2.3.3
 Flask-JWT-Extended==4.6.0
 gunicorn==21.2.0
-Flask-MySQL==1.5.2
\ No newline at end of file
+pytest-flask
+pytest
+PyMySQL
+PyMySQL[rsa]
```

Comment:
We don't strictly need this, but we will if we ever use caching_sha2_password or the like